src/gui/optionsdialog.cpp

```diff
@@ -218,8 +218,8 @@ OptionsDialog::OptionsDialog(QWidget *parent)
 
     m_ui->IpFilterRefreshBtn->setIcon(UIThemeManager::instance()->getIcon("view-refresh"));
 
-    m_ui->labelGlobalRate->setPixmap(Utils::Gui::scaledPixmap(UIThemeManager::instance()->getIcon(QLatin1String("slow_off")), this, 24));
-    m_ui->labelAltRate->setPixmap(Utils::Gui::scaledPixmap(UIThemeManager::instance()->getIcon(QLatin1String("slow")), this, 24));
+    m_ui->labelGlobalRate->setPixmap(Utils::Gui::scaledPixmapSvg(UIThemeManager::instance()->getIconPath(QLatin1String("slow_off")), this, 24));
```

Comment:
I think we should have only one `scaledPixmap()` that detects proper scaling method otherwise we have very inconvenient and unclear dependency since nothing in the code say about the icon type that is actually used.

Reply:
I agree but I have no idea whether it is possible. I presume it will be handled in a later PR.